tests/pbft_chain_test.cpp

```diff
@@ -85,7 +85,7 @@ TEST_F(PbftChainTest, pbft_db_test) {
 }
 
 TEST_F(PbftChainTest, proposal_block_broadcast) {
-  auto node_cfgs = make_node_cfgs<5>(3);
+  auto node_cfgs = make_node_cfgs<1>(3);
```

Comment:
So compared to @MatusKysel 's change of just adding more time to our weight before doing an equality check within the test, here we are just running this test in "real time" as opposed to 5x acceleration.    Thats fine, but I am curious as to what changed to make broadcast "slower"?